boot/boot.js

```diff
@@ -287,6 +287,136 @@ $tw.utils.decodeURIComponentSafe = function(s) {
 	return v;
 };
 
+/*************
+Code to make the permalink/permaview URLs more readable.
+(These comments are just to help reviewers; they will eventually be removed.)
+***************/
+
+
+/*************
+Helpers for encoding/decoding URIs.  **Note** that these are throw-away, only
+used to temporarily build `charMatch`, `spaceMatch`, `sentenceEnding`, and
+`sentenceTrailing`, all of which will eventually be hard-coded.
+***************/
```

Comment:
I think these 2 comments can be removed now. We know, what the PR is used for and the comments cannot stay in the merged PR.

Reply:
IMO the CONSTANTS should stay in the code. The js-engine will optimize the code. Hardcoded characters will be much harder to read and understand.